implementation/jobberknoll/api/int/routes/get-health-route.ts

```diff
@@ -21,7 +23,15 @@ export const getHealthRoute = createRoute({
   },
 });
 
-export function getHealthHandler(): RouteHandler<typeof getHealthRoute> {
-  // TODO @tchojnacki: Extract this logic to the application layer
-  return (c) => c.json({ status: "UP" }, 200);
+export function getHealthHandler(
+  getHealth: GetHealthUseCase,
+): RouteHandler<typeof getHealthRoute> {
+  return async (c) => {
+    const serviceHealth = expect(
+      await getHealth.invoke(null, uuid()),
+      "service health request should never fail",
+    );
+    const code = ["UP", "UNKNOWN"].includes(serviceHealth.status) ? 200 : 503;
+    return c.json(serviceHealth, code);
+  };
 }
```

Comment:
Looks like mindless boilerplate, and it currently is, however this will:
- let me gather health data from infrastructure (e.g. query Postgres for database availability), which was impossible below.
- add logs on health checks.